demo/index.html

```diff
@@ -1402,7 +1403,23 @@
               }
             }]
 
-            new hx.select('#graph').add(hx.graph({axes: axes}).classed('graph-example', true))
+            hx.select('#graph').add(hx.graph({axes: axes, legendEnabled: true}).classed('graph-example', true))
+
+            var data = {
+              title: "Hi",
+              segments: hx.range(6).map(function (i) {
+                return {
+                  name: "Segment " + i,
+                  size: i + 1,
+                  fillColor: hx.cycle(hx.theme.plot.colors, i)
+                }
+              })
+            }
+            hx.select('#pie').style('height', '100px').add(
+              hx.pieChart({legendEnabled: true, data: data}).style('width', '100%').style('height', '100%')
```

Comment:
This probably doesn't need changing for the PR but you _shouldn't_ need `width` here as `div`s are 100% width by default (`display: block`)
I'd also suggest that the `height: 100px` be applied directly to the newly created `hx.pieChart` as in the majority of cases that's how it would be used